src/components/core/Page/GlobalNavbar/LoggedInNav.tsx

```diff
@@ -47,46 +47,46 @@ function LoggedInNav() {
   }, []);
 
   return (
-    <Container>
-      <TextButton onClick={handleGalleryRedirect} text="My Gallery" />
+    <>
+      <NavElement>
```

Comment:
everything in the navbar should now be wrapped with NavElement; had diverging styles before this